CONTRIBUTING.md

```diff
@@ -59,23 +59,29 @@ Please note sources that require API authorization cannot be added to the catalo
 ```
 
 #### Add a GTFS Realtime source
-The easiest way to add a GTFS Realtime source is to use the operation `tools.operations.add_gtfs_realtime_source` through the Python interpreter or in your scripts. Provide the information about your source in the operation function to add your source. Note that even though `vehicle_positions_url`, `trip_updates_url` and `service_alerts_url`, you must provide at least one URL for your source to be valid.
+The easiest way to add a GTFS Realtime source is to use the operation `tools.operations.add_gtfs_realtime_source` through the Python interpreter or in your scripts. Provide the information about your source in the operation function to add your source.
 
 ```
 >>> add_gtfs_realtime_source(
+        entity_type=[$YOUR_SOURCE_ENTIY_TYPE],
         provider=$YOUR_SOURCE_PROVIDER_NAME,
-        static_reference=$OPTIONAL_STATIC_REFERENCE_NUMERICAL_ID,
-        vehicle_positions_url=$OPTIONAL_VEHICLE_POSITIONS_URL,
-        trip_updates_url=$OPTIONAL_TRIP_UPDATES_URL,
-        service_alerts_url=$OPTIONAL_SERVICE_ALERTS_URL,
-        name=$OPTIONAL_SOURCE_NAME
+        name=$OPTIONAL_SOURCE_NAME,
+        note=$OPTIONAL_SOURCE_NOTE,
+        static_reference=[$OPTIONAL_STATIC_REFERENCE_NUMERICAL_ID],
+        direct_download_url=$YOUR_SOURCE_DIRECT_DOWNLOAD_URL,
+        authentication_type=$CONDITIONALLY_REQUIRED_AUTHENTICATION_TYPE,
+        authentication_info_url=$CONDITIONALLY_REQUIRED_AUTHENTICATION_INFO_URL,
+        api_key_parameter_name=$CONDITIONALLY_REQUIRED_API_KEY_PARAMETER_NAME,
+        license_url=$OPTIONAL_LICENSE_URL,
```

Comment:
That's a helpful guideline! I was going based on the order in the README before. 

I also won't include status and features for now since #171 isn't merged.